test/controllers/announcement_controller_test.exs

```diff
@@ -11,19 +11,33 @@ defmodule ConstableWeb.AnnouncementControllerTest do
     insert(:announcement, title: "Awesome", user: user)
     insert(:announcement, title: "Lame", user: user)
 
-    conn = get conn, Routes.announcement_path(conn, :index, all: true)
+    conn = get(conn, Routes.announcement_path(conn, :index, all: true))
 
     assert html_response(conn, :ok) =~ "Awesome"
     assert html_response(conn, :ok) =~ "Lame"
   end
 
-  test "#index only my announcements are shown by default", %{conn: conn, user: user} do
+  test "#index shows user's announcements when user_id is set", %{conn: conn, user: user} do
```

Comment:
This is the new test that was added. The rest of the changes in this file were done by the auto-formatter.